### PR TITLE
basic code for search author

### PR DESCRIPTION
**Pull request by techbrothers3**

### CODE REVIEW

File: router/message.js [hunk 1]
```diff
@@ -77,6 +77,9 @@ router.get('/authorName',(req,res) => {
     },
   );
 });
+router.get('/author', (req, res) => {
```
Comment (mdshoeb01) [inline router/message.js:80]: add new line for new route.

File: router/message.js [hunk 2]
```diff
@@ -77,6 +77,9 @@ router.get('/authorName',(req,res) => {
...
+  res.send('hi');
```
Comment (mdshoeb01) [inline router/message.js:81]: make sure you update the response with proper data.

File: router/message.js (hunk 1 above)
Comment (techbrothers3) [reply router/message.js:80]: i will add in next pull request.

File: router/message.js (hunk 2 above)
Comment (techbrothers3) [reply router/message.js:81]: ok.

**Pull request merged**